Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Card.css';
+import CardImg from './CardBanner';
+import CardContent from './CardContent';
+
+const CardContainer = () => {
+    return (
+        <div onClick={myFunction} className="card-wrapper">
+            <CardImg />
+            <CardContent />
```

Comment:
I think the idea is that you want to pass all the values for the CardContent here (so like the h3, p, and a values that all get used in CardContent). So CardContent would look like `<CardContent h3Text="Get started with React" pText="React makes it painless to create interactive UIs. Design simple views for each state in your application."  />` .